### DISCUSSION TITLE
TVA et bounty

### DISCUSSION THREAD

**Discussion by Ga8** [noraj/BB-legal-FR#7] [General]
Merci pour tous vos efforts. 
Je vois une information qui me manque et qui  est trés difficile à trouver sur le web: 
-Est-ce que la TVA s'applique par dessus la récompense ou en dessous? 
C'est-a-dire est-ce que yes we hack par exemple quand ils rémunérent une prime de 200 euros est-ce qu'ils payent 240 euros quand on est soumis à la TVA ou ils payent 200 et au final c'est nous qui avons une perte seche de 40 euros. 

J'aurais aimé avoir cette info étant déjà freelance dev et soumis à la TVA . J'ai besoin de cette info pour calculer un business plan avant de me lancer.

**Comment by noraj** [comment]
Je n'ai pas la réponse à cette question, je n'ai pas trouvé de réponse dans les FAQ et CGV des plateformes françaises et ne suis pas personnellement assujetti à la TVA étant en dessous du seuil annuel. Je vais laisser cette question ouverte dans l'espoir qu'un hunter assujetti à la TVA puisse nous répondre.

**Comment by Ga8** [comment]
j'ai posé la question a yesWeHAck voila la réponse :

> 
> Bonjour,
> 
> Si cherchez des vulnérabilités sous le statut d’une entreprise et que vous êtes donc soumis à la TVA, vous devez vous déclarer en « 
> 
> company dans vos informations de facturation sur notre plateforme ( https://yeswehack.com/user/wallet/billing-information ). Vous accéderez à l’option permettant d’indiquer votre numéro de TVA et de cette manière, cette dernière sera alors automatiquement versée.
> 
> Vous trouverez plus d’informations sur notre helpcenter : 
> 
> https://helpcenter.yeswehack.io/billing-information


De ce que je comprends ca sera bien en + donc 200 euros + 40 euros de TVA. Je confirmerai quand j'aurai un peu de temps pour trouver une disclosure. (assez vite j'espère)

**Comment by Ga8** [comment]
Yes la TVA est bien versé **au dessus** de la prime. Du style 100€  ==> YesWeHack verse 120 € quand tu es soumis à la TVA. Les autres plateforment n"étant pas francaise je ne crois pas qu'on soit soumis a la TVA avec eux.

**Reply by noraj** [reply]
Ce qui veut dire que YWH affiche le prix des primes HT.
